src/components/Details/Details.css

```diff
@@ -0,0 +1,44 @@
+:root {
+  --dark-blue: #1a3871;
+  --white: rgb(255, 255, 255);
+  --lightblue: #4872c2;
+}
+
+body {
+  background-color: var(--lightblue);
+  color: var(--white);
+}
+
+.detail-data {
```

Comment:
- [ ] It  would be better if we address the issue of padding in our `.detail-data` CSS class. It is important we follow the design as seen in the guideline below
![Screenshot from 2022-11-29 09-42-22](https://user-images.githubusercontent.com/46388113/204496041-f694f2d7-8495-40ab-934f-01390d5185f9.png)
![Screenshot from 2022-11-29 09-42-54](https://user-images.githubusercontent.com/46388113/204496055-040928cd-7256-4b27-b604-a8df24f43a78.png)

Here we mean the space between the text and the text-wrapper